src/meta/src/manager/sink_coordination/coordinator_worker.rs

```diff
@@ -252,10 +257,12 @@ impl CoordinatorWorker {
         &mut self,
         mut coordinator: impl SinkCommitCoordinator,
     ) -> anyhow::Result<()> {
-        coordinator.init().await?;
+        let log_store_rewind_start_epoch = coordinator.init().await?;
         loop {
-            let (handle_id, vnode_bitmap, epoch, metadata) =
-                self.handle_manager.next_commit_request().await?;
+            let (handle_id, vnode_bitmap, epoch, metadata) = self
+                .handle_manager
+                .next_commit_request(log_store_rewind_start_epoch)
+                .await?;
```

Comment:
The `log_store_rewind_start_epoch` here is a const, while in `next_commit_request` we can't rewind to `log_store_rewind_start_epoch` for each new worker (considering online scaling, a new worker will join in the middle).

Reply:
cc @wenym1 It doesn't appear to be a correctness issue, as the coordinator will be terminated after scaling.